.github/workflows/release.yml

```diff
@@ -1,9 +1,34 @@
-name: Publish JS gRPC bindings
+name: Release
 on:
   release:
-    types: [published]
+    types: [created]
 jobs:
-
+  publish_docker:
+    name: Publish Docker Compose Setup
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+      - name: Get latest tag
+        id: latesttag
+        uses: "WyriHaximus/github-action-get-previous-tag@master"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      - name: Update powergate image version
+        run: sed -i.bak 's/latest/${{steps.latesttag.outputs.tag}}/g' docker/powergate-image.yaml
```

Comment:
Updates the powergate image tag to use in the compose setup to be the release tag.